deps.edn

```diff
@@ -44,4 +44,7 @@
             :extra-deps
             {babashka/pods {#_#_:local/root "/Users/borkdude/Dropbox/dev/clojure/babashka/babashka.pods"
                             :git/url "https://github.com/babashka/babashka.pods"
-                            :sha "57cc95a2e2193a4bfb8505d44acaca900279369a"}}}}}
+                            :sha "57cc95a2e2193a4bfb8505d44acaca900279369a"}}}
+           :eastwood
+           {:main-opts ["-m" "eastwood.lint" {:source-paths ["src" "test"]
+                                              :exclude-linters [:unused-ret-vals]}]}}}
```

Comment:
`:unused-ret-vals` almost could be enabled, but there's a false positive.

It would be nice to be able to mark arbitrary defns in Eastwood as side-effectful to avoid this. As you know that is done in var-info.edn, but that is an entirely closed set.